CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
 
+# Determine OS
+exec_program(uname OUTPUT_VARIABLE SYSTEM_NAME)
```

Comment:
Let's move this to a function in utils.cmake that returns the SYSTEM_NAME. Example: https://cmake.org/pipermail/cmake/2008-November/025423.html